File: CSHARP/CSHARP/Model/PartijaDvaIgraca.cs
using System;
namespace CSHARP.Model
{
    public class PartijaDvaIgraca: Partija
    {
        public override string ToString()
        {
            var r = getRezultat();
            return "Partija DVA IGRAČA, igra gotova: " + isIgraGotova() + ", " + igraci[0] + ": " +
                    r.prvi +
                    " | " + igraci[1] + ": " + r.drugi;
        }
    }
}

File: Kotlin/src/main/kotlin/model/PartijaDvaIgraca.kt
package model

/**
 * @author Matija Skrletović
 **/

open class PartijaDvaIgraca(
    override var doKolikoSeIgra: Int, override val lokacija: Lokacija,
    override val unosi: Igrac, override val mjesanja: List<Mjesanje>,
    override val igraci: MutableList<Igrac?>
) : Partija() {

    override fun toString(): String {
        val rezultat = getRezultat()
        return "Partija DVA IGRAČA, igra gotova: " + isIgraGotova() + ", " + igraci[0] + ": " +
                rezultat.prvi +
                " | " + igraci[1] + ": " + rezultat.drugi
    }
}